spec/models/test_track/ab_configuration_spec.rb

```diff
@@ -86,7 +86,7 @@
         with_rails_env 'production' do
           expect do
             described_class.new initialize_options.merge(split_name: :not_a_real_split)
-          end.not_to raise_error("unknown split: not_a_real_split. You may need to run rake test_track:schema:load")
+          end.to raise_error("unknown split: not_a_real_split.")
```

Comment:
The latest version of rspec didn't like this style: ```Using `expect { }.not_to raise_error(message)` risks false positives, since literally any other error would cause the expectation to pass```